.github/workflows/containers.yml

```diff
@@ -23,19 +32,26 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Set extra environment variables
+        run: |
+          case "${ARCHITECTURE}" in
+            386)   echo MACHINE="i386" >> "$GITHUB_ENV";;
```

Comment:
If you change `386` to `i386`, you can just have a default case to set `MACHINE` to `$ARCHITECTURE`.